components/admin_console/team_channel_settings/group/group_users/group_users_groups.tsx

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {Tooltip} from 'react-bootstrap';
+import {FormattedMessage} from 'react-intl';
+
+import {UserProfile} from 'mattermost-redux/types/users';
+import {Group} from 'mattermost-redux/types/groups';
+
+import {t} from 'utils/i18n';
+import OverlayTrigger from 'components/overlay_trigger';
+
+type ProfileWithGroups = Partial<UserProfile & {
+    groups: Partial<Group>[];
+}>;
+
+interface GroupUsersGroupsProps {
+    user: ProfileWithGroups;
+}
+export default class GroupUsersGroups extends React.PureComponent<GroupUsersGroupsProps, {}> {
```

Comment:
Perhaps this could be done as a functional component instead? We're trying to start using those wherever we can since they can contain hooks and since they're supposed to perform better in the future.